src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/custom.py

```diff
@@ -334,10 +335,17 @@ def _get_site_credential(client, resource_group, name):
     return (creds.publishing_user_name, creds.publishing_password)
 
 def _stream_trace(streaming_url, user_name, password):
-    import pycurl
-    import certifi
-    c = pycurl.Curl()
-    c.setopt(c.URL, streaming_url)
-    c.setopt(pycurl.CAINFO, certifi.where())
-    c.setopt(c.USERPWD, '{}:{}'.format(user_name, password))
-    c.perform()
+    import urllib3
+
+    urllib3.disable_warnings()
```

Comment:
Which SSL warnings? Suppressing potential security related warnings is scary to me...